pom.xml

```diff
@@ -66,4 +66,22 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-source-plugin</artifactId>
```

Comment:
uhm... you depends on the source jar but you do not deploy the source jar on install... so it fails. I just add it so mvn install deploy both the jar and the source jar.